### DISCUSSION TITLE
新人入站，请多指教。放张简图

### DISCUSSION THREAD

**Discussion by Jerryemm** [share121/inter-knot#3102] [其他]
![b127bb6d8d5c7d397572f92034467453](https://github.com/user-attachments/assets/75c81d31-fa68-4937-83aa-fdea559ef3cf)

**Comment by Jerryemm** [comment]
我太爱简了

**Comment by warning-Kirov-reporting** [comment]
豪堪<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >
